src/js/media-container.js

```diff
@@ -166,9 +166,9 @@ class MediaContainer extends MediaChromeHTMLElement {
     // Toggle play/pause with clicks on the media element itself
     this._mediaClickPlayToggle = e => {
       if (media.paused) {
-        this.dispatchMediaEvent(mediaUIEvents.MEDIA_PLAY_REQUEST);
+        this.dispatchMediaEvent(MediaUIEvents.MEDIA_PLAY_REQUEST);
```

Comment:
`dispatchMediaEvent` is gone now, right?

Reply:
ah missed one! yeah it is/should be. Good 🦅 👀